src/main/java/com/launchacademy/giantleap/models/Review.java

```diff
@@ -0,0 +1,62 @@
+package com.launchacademy.giantleap.models;
+
+import lombok.Data;
+import org.hibernate.validator.constraints.Range;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.stereotype.Service;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+
+@Entity
+@Table(name = "reviews")
+@Data
+public class Review {
+    @Id
+    @SequenceGenerator(name="review_generator", sequenceName="reviews_id_seq", allocationSize = 1)
+    @GeneratedValue(strategy= GenerationType.SEQUENCE, generator="review_generator")
+    @Column(name="id", nullable=false, unique=true)
+    private Integer id;
+
+    @NotBlank
+    @Column(name="title")
+    private String title;
+
+    @NotNull
+    @Column(name="date", columnDefinition = "DATE")
+    private Date date;
+
+    @NotBlank
+    @Column(name="review")
+    private String review;
+
+    @NotNull
+    @Column(name="cleanliness")
+    @Range(min=1, max=5)
+    private Integer cleanliness;
+
+    @NotNull
+    @Column(name="on_time")
+    @Range(min=1, max=5)
+    private Integer onTime;
+
+    @NotNull
+    @Column(name="capacity")
+    @Range(min=1, max=5)
+    private Integer capacity;
+
+    @NotNull
+    @Column(name="overall_rating")
+    @Range(min=1, max=5)
+    private Integer overallRating;
+
+    @ManyToOne
+    @JoinColumn(name="station_id")
+    private Station station;
+
+    @ManyToOne
+    @JoinColumn(name="user_id")
+    private User user;

```

Comment:
You need to set up the OneToMany relationships on the other models.

Realistically if a Review belongs to a user *and* a station then there should be a join table called user_station_reviews. 
I'll approve this as is for now, but please either reduce to one relation, or you'll have to implement @ManyToMany